### PR TITLE
Separate container for each device documentation and improved macOS steps

### PR DESCRIPTION
**Pull request by PatrickDuncan**

### CODE REVIEW

**Comment by butomo1989** [comment]
Hi @PatrickDuncan , 

It looks good to me. But it is possible that one docker-appium container can handle multiple devices. Could you give me a hint / use case why you need to have one device for one container?

**Comment by PatrickDuncan** [comment]
Hey @butomo1989 ,

First of all thank you for all your work on making this image! We've designed an architecture that needs  each device on its own Docker container. We thought it would be useful to document outside of our internal dev team.

However, I do think you should accept the `adb kill-server` section of this PR undoubtedly.

**Comment by SrinivasanTarget** [comment]
@PatrickDuncan Any particular reason why do we need n docker containers for n real devices connected to a machine. Latest version of appium needs one server to handle n devices. Is there any issues that you face in docker so you suggest the old fashion way of appium?

**Comment by PatrickDuncan** [comment]
Hey @SrinivasanTarget ,

Thanks for responding! Correct me if I'm wrong, but when a container is created with the option to connect to the Hub it creates 1 node config file that encompasses every Android device that is currently connected. So when a device is removed or a new device is connected the Node on the Hub will only show the initial devices.

That approach will not work for us because we're always connected to a Hub and devices will be connected/disconnected from the machine constantly. We have a mechanism that initiates an Appium container automatically when the device is connected using the contents of this pull request. (We also have a mechanism to stop/remove containers when a device is disconnected).

Hope that that clarifies what we're doing @SrinivasanTarget @butomo1989 !

File: README.md [hunk 1]
```diff
@@ -98,7 +103,18 @@ Each time, you will (re)create container, connected to container devices will as
 For example:
 ```
 $ docker run --privileged -d -p 4723:4723 -v ~/.android:/root/.android -v /dev/bus/usb:/dev/bus/usb --name container-appium appium/appium
-``` 
+```
+
+## Connect to Multiple Android Devices on Linux
+
+The parameter `-v /dev/bus/usb:/dev/bus/usb` doesn't work for multi-device because it siphons off every usb device to a single container. Every device will to be connected to the same Appium server which doesn't work well in practice.
```
Comment (SrinivasanTarget) [inline README.md]: Any reason why do you suggest that ` same appium server which doesnt work well in practice`? We have been running multiple tests on single appium server hooked to multiple devices without any issues for along time.

Comment (PatrickDuncan) [reply README.md]: Hi, I was referring to plugging in new devices with the Hub and the old Appium issues, but I do agree that the paragraph should be removed. I'll remove it and fix the conflict soon.

**Comment by PatrickDuncan** [comment]
Hello again @butomo1989 @SrinivasanTarget ,

Thanks for merging #76 ! Let me know if anything else needs to be changed for this pull request

File: README.md (hunk 1 above)
Comment (SrinivasanTarget) [reply README.md]: Sure fix this

Comment (PatrickDuncan) [reply README.md]: Hey I fixed it yesterday, please look at `Files changed`.

**Pull request merged**